LongWarOfTheChosen/Config/XComLW_UI.ini

```diff
@@ -66,3 +66,10 @@ SHOW_REGION_TOOLTIPS_CTRL = false			; Shows the region tooltips when using a con
 
 [LW_Overhaul.UIStrategyMapItem_Mission_LW]
 SHOW_MISSION_TOOLTIPS_CTRL = true			; Shows tooltips for missions when using a controller. These tooltips are the same as what you would see on the bottom UI bar when using a mouse & keyboard.
+
+[LW_Overhaul.UIPersonnel_SoldierListItem_LW]
+CI_STANDARD_COLOUR = "0xbf1e2e"				; Standard combat intelligence text colour.
+CI_ABOVEAVERAGE_COLOUR = "0xe69831"			; Above average combat intelligence text colour.
+CI_GIFTED_COLOUR = "0xfdce2b"				; Gifted combat intelligence text colour.
+CI_GENIUS_COLOUR = "0x53b45e"				; Genius combat intelligence text colour.
+CI_SAVANT_COLOUR = "0x27aae1"				; Savant combat intelligence text colour.
```

Comment:
Thanks for doing this, but would you mind making a couple of changes, please?
1. Use American English spelling in code (`COLOR` rather than `COLOUR`)
2. I'd prefer `COMINT` (or `COM_INT` if you want to be very consistent) because `CI` is an overloaded acronym (Continuous Integration and Covert Infiltration for eaxmple) and the enum values start `eComInt`